index.html

```diff
@@ -285,5 +285,6 @@ <h2>About me</h2>
       type="module"
       src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"
     ></script>
+    <script src="node.js"></script>
```

Comment:
- [ ] The mobile is not responsive, I have to move the page to the right to see the close button please make sure the mobile responsive on the mobile screen

- [ ] please make sure to make the bottom of the menu rounded as in the Figma design 


- Yours 
 
<img width="299" alt="Screenshot 2023-03-06 at 15 28 33" src="https://user-images.githubusercontent.com/61790587/223140028-54fb67d8-73fc-4402-9537-3938c7a00bab.png">
- Design 
<img width="417" alt="Screenshot 2023-03-06 at 14 39 00" src="https://user-images.githubusercontent.com/61790587/223140054-45bbd0a8-7ee2-488b-b3c1-ad26eef89971.png">